demo/app/views/layouts/application.html.erb

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <html class="no-js" lang="<%= I18n.locale %>">
 <head>
+  <%= display_meta_tags site: "Citizens Advice", reverse: true, separator: "-", title: @page_title, description: @page_description %>
```

Comment:
This shouldn't need you to explicit pass the variables in, but doesn't do any harm for just the demo app.